python/cuml/tests/test_hdbscan.py

```diff
@@ -166,7 +164,6 @@ def test_hdbscan_blobs(
     )
 
     cuml_agg = HDBSCAN(
-        verbose=logger.level_enum.info,
```

Comment:
This was a previously uncaught user-bug since `logger.level_enum` is an `int` subclass. It wouldn't do what was intended though (setting info-level logs). Dropping this to do what was actually intended, since the default log level is info anyway.